src/node_version.h

```diff
@@ -24,12 +24,12 @@
 
 #define NODE_MAJOR_VERSION 10
 #define NODE_MINOR_VERSION 13
-#define NODE_PATCH_VERSION 1
+#define NODE_PATCH_VERSION 2
```

Comment:
I don't think this should be changed under the subsequent `Now working on v10.13.2` commit that also changes `NODE_VERSION_IS_RELEASE` back to `0`.